src/app/pages/poll/poll.component.ts

```diff
@@ -62,13 +66,17 @@ export class PollComponent {
     this.onUserDataChanged.subscribe(user => {
       this.userData = user;
       // Check if pollId is already available
-      if (this.pollId) this.loadPoll();
+      if (this.pollId) {
+        this.loadPoll();
+      }
     });
     // Subscribe to active route
-    this.activeRoute.params.subscribe( params => {
+    this.activeRoute.params.subscribe(params => {
```

Comment:
Love that you don't need to use `(params)` anymore (with parantheses). Did you change this in the TypeScript linter settings?

Reply:
No didn't change the linter setting. But the problem was in the SonaQube settings, not the Typescript linter settings

Reply:
ok